packages/math-input/src/components/tabbar/item.tsx

```diff
@@ -75,70 +76,146 @@ function imageTintColor(
     return color.offBlack64;
 }
 export type ItemState = "active" | "inactive" | "disabled";
-type Props = {
+type ArrowKeyTabItemProps = {
     onClick: () => void;
     itemState: ItemState;
     itemType: KeypadPageType;
+    focus: boolean;
+    setFocus: React.Dispatch<React.SetStateAction<number>>;
+    index: number;
 };
 
-class TabbarItem extends React.Component<Props> {
-    render(): React.ReactNode {
-        const {onClick, itemType, itemState} = this.props;
-        return (
-            <Clickable
-                onClick={onClick}
-                disabled={itemState === "disabled"}
-                aria-label={itemType}
-                style={styles.clickable}
-                aria-selected={itemState === "active"}
-                role="tab"
-            >
-                {({hovered, focused, pressed}) => {
-                    const tintColor = imageTintColor(
-                        itemState,
-                        hovered,
-                        focused,
-                        pressed,
-                    );
+type TabItemProps = {
+    onClick: () => void;
+    itemState: ItemState;
+    itemType: KeypadPageType;
+};
+
+export function ArrowKeyTabbarItem(
```

Comment:
The render functions for `ArrowKeyTabbarItem` and `TabbarItem` are almost 99% identical. Regularly we run into issues in Perseus when copy/pasting code because the two copies grow and diverge and after awhile the reasons for copying the code gets lost to time.

I think this might be an opportunity to consolidate these two components and make one, flexible component or to use composition to separate the focus logic from the component logic.